lib/tori/backend/filesystem.rb

```diff
@@ -39,6 +39,8 @@ def read(filename)
         ::File.read(path(filename), mode: 'rb')
       end
 
+      private
+
```

Comment:
I think that public is more better because to easily test and also want to know path so this is *filesystem*.